app/passport/views.py

```diff
@@ -103,3 +105,80 @@ def passport(request, pattern):
         ],
     }
     return JsonResponse(context)
+
+
+def verifiable_credential(request):
+    passport = None
+
+    player = request.GET.get("coinbase")
+    network = request.GET.get("network")
+
+    if not request.user.is_authenticated:
+        return JsonResponse({"status": "error", "msg": "You must login"})
```

Comment:
^ this can be removed since we are using the decorator

Reply:
Resolved in https://github.com/gitcoinco/web/pull/9032/commits/91afdb82d0b077c0955311ec642a210ea34ab0af